samples/wifi/shutdown/README.rst

```diff
@@ -57,6 +57,20 @@ The following is an example of the CLI command to demonstrate Wi-Fi shutdown:
 
    west build -b nrf7002dk_nrf5340_cpuapp
 
+Disable auto-start of the Wi-Fi driver
+--------------------------------------
+
+The Wi-Fi network interface is automatically brought up when the Wi-Fi driver is initialized by default.
+But you can disable this behavior by setting the ``CONFIG_WIFI_INIT_AUTO_START`` configuration option to ``n``.
+
+Please use additional configuration file ``overlay-no_wifi_auto_start.conf`` to disable auto-start of the Wi-Fi driver:
+
+.. code-block:: console
+
+   west build -b nrf7002dk_nrf5340_cpuapp -DEXTRA_CONF_FILE=overlay-no_wifi_auto_start.conf
```

Comment:
What is the benefit of using a single line overlay instead of passing `-DCONFIG_NRF_WIFI_IF_AUTO_START=n` directly?

Reply:
Left as a place holder for further configs, but can remove .